tool/gravity/cli/logging.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cli
+
+import (
+	"fmt"
+	"log/syslog"
+	"strings"
+
+	"github.com/gravitational/gravity/lib/constants"
+	"github.com/gravitational/gravity/lib/utils"
+
+	"github.com/gravitational/trace"
+)
+
+// LogCLIRunning writes the running cmd as a log entry into the system journal
+// with the gravity-cli tag.
+func LogCLIRunning(cmd string) {
+	entry := fmt.Sprintf("[RUNNING]: %s", cmd)
```

Comment:
Also, what about my suggestion to assign a unique uuid4 tag to each command run and log it as well? So we can correlate start/finish of the commands?

Reply:
The journal entries already include the process ID which can be used to track start/finish. Is there any benefit to adding a UUID4 tag in this case?

We can track the command with the `_PID` flag. Example:
```
[vagrant@node-1 ~]$ sudo journalctl -t gravity-cli _PID=29778
-- Logs begin at Fri 2020-07-10 20:26:59 UTC, end at Sat 2020-07-11 14:36:30 UTC. --
Jul 11 14:05:37 node-1 gravity-cli[29778]: [RUNNING]: /usr/bin/gravity install --debug --token "*****"
Jul 11 14:05:37 node-1 gravity-cli[29778]: [FAILURE]: /usr/bin/gravity install --debug --token "*****": [ERROR]: install token is too short, min length is 6
```

Reply:
Good point. No need for additional ids then.